app/jobs/regular/bulk_grant_trust_level.rb

```diff
@@ -0,0 +1,21 @@
+module Jobs
+
+  class BulkGrantTrustLevel < Jobs::Base
+
+    def execute(args)
+      trust_level = args[:trust_level]
+      user_ids = args[:user_ids]
+
+      raise Discourse::InvalidParameters.new(:trust_level) if trust_level.blank?
+      raise Discourse::InvalidParameters.new(:user_ids) if user_ids.blank?
+
+      User.where(id: user_ids).find_each do |user|
+        if user.trust_level < trust_level
```

Comment:
Maybe just extract this logic below and https://github.com/discourse/discourse/blob/master/app/models/group_user.rb#L56-L63 into a concern or Service object?